setup.py

```diff
@@ -44,6 +44,7 @@
         'nc-time-axis<1.3.1',  # needed by iris.plot, issues with matplotlib 3.4 and 1.3.1
         'netCDF4',
         'numpy',
+        'openpyxl',
```

Comment:
to be sure the env solves well you can just look at the [Circle test](https://app.circleci.com/pipelines/github/ESMValGroup/ESMValTool/5529/workflows/a74015dd-34b7-4e35-a49c-d3f78c880e98/jobs/49209) that built succesfully a complete environment with the new deps list - all good there :+1: